src/index.js

```diff
@@ -0,0 +1,233 @@
+class Helper {
+  static getSelectedSky() {
```

Comment:
Ideally this would return a class name instead of a path to a file and then you could have that class name set the background image to your particular image.  

This enforces [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).